src/fetchAndTransformArticle.js

```diff
@@ -32,18 +33,24 @@ export async function transformArticle(
     lang
   );
 
+  const title = markdownToHtmlString(article.title.title ?? '');
+
+  const introduction = markdownToHtmlString(
+    article?.introduction?.introduction ?? ''
+  );
+
+  const metaDescription = markdownToHtmlString(
+    article?.metaDescription?.metaDescription ?? ''
+  );
```

Comment:
Markdown i introduction rendres i ndla-frontend/editorial-frontend allerede, så denne trengs vel ikkje være med. Om ikkje du fjerner den andre renderinga då.

metaDescription vises ikkje fram i sidene kun i metatags for artikkelen og det er rein tekst og ikkje html-formatert, så markdown der gir ikkje så masse meining. Einaste plass metadescription vises i ndla-frontend er i søkeresultat, og då hentes det fra search-api.